metrics-clojure-ring/src/metrics/ring/expose.clj

```diff
@@ -77,24 +78,45 @@
 (defn- render-metric [[metric-name metric]]
   [metric-name (render-to-basic metric)])
 
+(defn- placeholder? [c]
+  (#{"" "*"} c))
 
 ;;
 ;; API
 ;;
 
+(defn filter-matches? [kn filter]
+  (every? identity
```

Comment:
It might be slightly clearer to use `true?` here but I don't feel strongly about this.